statistics_service/native/src/impl/StatisticsService.cpp

```diff
@@ -1,164 +1,234 @@
+    return ScopedAStatus::ok();
+}
 
-        ::ndk::ScopedAStatus StatisticsService::getAverageAmbientTemperature(float *value_return)
-        {
-            CallbacksId id = CallbacksId::AverageAmbientTemperature;
-            value_return = getValue(id);
-            return ndk::ScopedAStatus::ok();
-        }
+ScopedAStatus StatisticsService::GetAverageAmbientTemperature(float *value_return) {
+    *value_return = GetValue(CallbacksId::AverageAmbientTemperature);
+    return ScopedAStatus::ok();
+}
 
-        ::ndk::ScopedAStatus StatisticsService::getMaxCpuTemperature(float *value_return)
-        {
-            CallbacksId id = CallbacksId::MaxCpuTemperature;
-            value_return = getValue(id);
-            return ndk::ScopedAStatus::ok();
-        }
+ScopedAStatus StatisticsService::GetMaxCpuTemperature(float *value_return) {
+    *value_return = GetValue(CallbacksId::MaxCpuTemperature);
+    return ScopedAStatus::ok();
+}
 
-        ::ndk::ScopedAStatus StatisticsService::getMaxGpuTemperature(float *value_return)
-        {
-            CallbacksId id = CallbacksId::MaxGpuTemperature;
-            value_return = getValue(id);
-            return ndk::ScopedAStatus::ok();
-        }
+ScopedAStatus StatisticsService::GetMaxGpuTemperature(float *value_return) {
+    *value_return = GetValue(CallbacksId::MaxGpuTemperature);
+    return ScopedAStatus::ok();
+}
 
-        ::ndk::ScopedAStatus StatisticsService::getMaxAmbientTemperature(float *value_return)
-        {
-            CallbacksId id = CallbacksId::MaxAmbientTemperature;
-            value_return = getValue(id);
-            return ndk::ScopedAStatus::ok();
-        }
+ScopedAStatus StatisticsService::GetMaxAmbientTemperature(float *value_return) {
+    *value_return = GetValue(CallbacksId::MaxAmbientTemperature);
+    return ScopedAStatus::ok();
+}
 
-        float StatisticsService::readTemperatures() {
-            std::vector<float> values;
-
-            if(current_line_to_read > TOTAL_NR_LINES)
-                current_line_to_read = 1;
-
-            std::fstream file("temperatures.csv");
-
-            if (file.is_open())
-            {
-                std::string line;
-                for(int i = 1; getline(file, line); ++i)
-                {
-                    if(i == current_line_to_read) {
-                        std::stringstream stream(line);
-                        std::string value;
-
-                        while(getline(stream, value, ','))
-                        {
-                            std::stringstream token(value);
-                            float myFloat;
-                            token >> myFloat;
-                            values.push_back(myFloat);
-                        }
-
-                        ++current_line_to_read;
-                        file.close();
+void StatisticsService::ReadTemperatures() {
+    std::ifstream file("/vendor/etc/temperatures.csv");
+
+    if (file.is_open()) {

```

Comment:
o chestie de stil, minor, dar eu prefer "early exit" atunci cand apare o eroare. Aplicand acest stil, vei evita functii ce au linii foarte lungi, cu indent mare.
e.g.
```suggestion
    if (!file.is_open()) {
        // log error
        return;
    }

```